### PR TITLE
Update prod configuration

### PR DESCRIPTION
**Pull request by wowkalucky**

### CODE REVIEW

File: bridge_adaptivity/Dockerfile [hunk 1]
```diff
@@ -11,6 +11,5 @@ ADD . /bridge_adaptivity/
 RUN pip install -U pip
 RUN pip install -Ur requirements.txt
 
-# Collect static media:
+# Make static media dir:
 RUN mkdir -p /www/static
-RUN python manage.py collectstatic --noinput --settings=config.settings.base
```
Comment (wowkalucky) [inline bridge_adaptivity/Dockerfile]: Moved `collectstatic` to `prod_run.sh`

File: bridge_adaptivity/docker-compose.yml [hunk 2]
```diff
@@ -31,6 +31,8 @@ services:
       - "443:443"
     volumes_from:
       - bridge
+    volumes:
+      - /etc/nginx/ssl/:/etc/nginx/ssl/
```
Comment (wowkalucky) [inline bridge_adaptivity/docker-compose.yml]: In order to perform SSL certs configuration.

File: bridge_adaptivity/requirements.txt [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # install base requirements
 -r requirements_base.txt
+-r requirements_local.txt  # FIXME(wowkalucky): should add it here to satisfy debug_toolbar module import (somewhere)
```
Comment (wowkalucky) [inline bridge_adaptivity/requirements.txt]: This is Strange Thing...

Comment (idegtiarov) [reply bridge_adaptivity/requirements.txt]: Please remove included requirements_local from the requirements file it looks confusing.

Comment (idegtiarov) [reply bridge_adaptivity/requirements.txt]: Please remove included requirements_local from the requirements file it looks confusing.

**Pull request merged**